xml/libvirt_overview.xml

```diff
@@ -19,65 +19,393 @@
  <para>
-  The communication between the virtualization solutions (&kvm;, &xen;)
-  and the libvirt API is managed by the &libvirtd; daemon. It needs to run
-  on the &vmhost;. libvirt client applications such as virt-manager, possibly
-  running on a remote machine, communicate with &libvirtd; running on the
-  &vmhost;, which services the request using native hypervisor APIs. Use the
-  following commands to start and stop &libvirtd; or check its status:
+  A libvirt deployment for accessing KVM or Xen requires one or more daemons
+  to be installed and active on the host. libvirt provides two daemon deployment
+  options: monolithic or modular daemons. By default the monolithic daemon is
+  enabled, but a deployment can be switched to modular daemons by managing the
+  corresponding systemd service files.
  </para>
- <screen>&prompt.sudo;systemctl start libvirtd
 
-&prompt.sudo;systemctl status libvirtd
-libvirtd.service - Virtualization daemon
-Loaded: loaded (/usr/lib/systemd/system/libvirtd.service; enabled)
-Active: active (running) since Mon 2014-05-12 08:49:40 EDT; 2s ago
-[...]
-
-&prompt.sudo;systemctl stop libvirtd
-
-&prompt.sudo;systemctl status libvirtd
-[...]
-Active: inactive (dead) since Mon 2014-05-12 08:51:11 EDT; 4s ago
-[...]</screen>
  <para>
-  To automatically start &libvirtd; at boot time, either activate it using the
-  &yast; <guimenu>&ycc_runlevel;</guimenu> module or by entering the following
-  command:
+  The modular daemon deployment is useful in scenarios where minimal libvirt
+  support is needed. For example if virtual machine storage and networking is
+  not provided by libvirt, the various libvirt-daemon-driver-storage and
+  libvirt-daemon-driver-network packages are not required. Kubernetes is an
+  example of an extreme case, where it handles all networking, storage, cgroups
+  and namespace integration, etc. Only the libvirt-daemon-driver-qemu package,
+  providing virtqemud, needs to be installed. Modular daemons allow configuring
+  a custom libvirt installation containing only the components required for
+  the use-case.
  </para>
- <screen>&prompt.sudo;systemctl enable libvirtd</screen>
+ 
+ <sect1 xml:id="libvirt-monolithic-daemon">
+  <title>Starting and stopping the monolithic daemon</title>
+
+  <para>
+   The monolithic daemon is known as libvirtd and is configured via
+   /etc/libvirt/libvirtd.conf. libvirtd is managed with several systemd unit
+   files:
+  </para>
+
+  <itemizedlist>
+   <listitem>
+    <para>
+     <emphasis>libvirtd.service</emphasis> - The main systemd unit file for
+     launching libvirtd. It is recommended to configure libvirtd.service to
+     start on boot if VMs are also configured to start on host boot.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     <emphasis>libvirtd.socket</emphasis> - The unit file corresponding to the
+     main read-write UNIX socket /var/run/libvirt/libvirt-sock. This socket is
+     recommended to be started on boot by default.
```

Comment:
what does it mean for a user to "start" a UNIX socket?

Reply:
I reworded the text for the various socket unit files.